chanzuckerberg/single-cell-data-portal
# feat(wmg): citation tool - 4608

## Reason for Change

Provide an easy citation button for Gene Expression users

[Add easy citation to Gene Expression 4608
](https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/gh/chanzuckerberg/single-cell-data-portal/4608)

## Changes

- added Citation Button and Notification. The button, when clicked, will copy Citation text+link to the user's clipboard.

## Testing steps

- In Gene Expression, select at least one gene - the Citation Button should light up
- Click Citation button
- A Notification will pop up indicating that Citation was copied to clipboard
- Pasting from clipboard should have the Citation link
- The Notification should disappear after 5 seconds

## Notes for Reviewer


https://github.com/chanzuckerberg/single-cell-data-portal/assets/7976579/c41dd40e-ee8b-46d4-8e49-74db76bcfabc
